examples/trials/mnist-tfv2/mnist.py

```diff
@@ -116,7 +116,7 @@ def main(params):
         x_train,
         y_train,
         batch_size=params['batch_size'],
-        epochs=10,
+        epochs=5,
```

Comment:
FYI, 10 epochs may make windows IT timeout.
Since this is no longer used in tutorial, I guess it's okay to reduce it.